hub-backend/src/main/java/org/panda_lang/hub/authentication/AuthenticationEndpointsController.kt

```diff
@@ -43,7 +42,7 @@ internal class AuthenticationEndpointsController(
     fun authenticateUser(@RequestBody dto: SignInDto): ResponseEntity<Map<String, Any?>> {
         val authentication = authenticationManager.authenticate(UsernamePasswordAuthenticationToken(dto.name, dto.password))
         SecurityContextHolder.getContext().authentication = authentication
-        return ResponseEntity.ok(Collections.singletonMap<String, Any?>("access_token", authenticationTokenCreator.create(authentication)))
-    }
 
-}
\ No newline at end of file
+        return ResponseEntity.ok(mapOf("access_token" to authenticationTokenCreator.create(authentication)))
```

Comment:
Java's Collections.singletonMap may be replaced with a `mapOf(x to y)`. This also removes the explicit generic arguments.